README.md

```diff
@@ -123,24 +139,58 @@ To initiate an upgrade process, a user has to update the `spec.image.tag` field
 
 After the `tag` field is updated, the StatefulSet will detect the version change and it will trigger the upgrade process.
 
-To detect whether a `ZookeeperCluster` upgrade is in progress or not, check the output of the command `kubectl get sts <name> -o yaml`. The output of this command contains the following entries
+To detect whether a `ZookeeperCluster` upgrade is in progress or not, check the output of the command `kubectl describe zk`. Output of this command should contain the following entries
 
 ```
+$ kubectl describe zk
+
 status:
-  collisionCount: 0
-  currentReplicas: 2
-  currentRevision: example-75d4b645c8
-  observedGeneration: 4
-  readyReplicas: 3
-  replicas: 3
-  updateRevision: example-5d9dfdb787
+Last Transition Time:    2020-05-18T10:25:12Z
+Last Update Time:        2020-05-18T10:25:12Z
+Message:                 0
+Reason:                  Updating Zookeeper
+Status:                  True
+Type:                    Upgrading
+```
+
+If the Status value for clustercondition type `Upgrading` is True, it signifies that the cluster is in Upgrading state.
+
+Additionally, The output for the `$ kubectl get zk` will also have the Desired version set to the Image Tag the zookeepercluster is trying to Upgrade to.
+
+```
+$ kubectl get zk
+
+NAME         REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT     EXTERNAL ENDPOINT   AGE
+example        3          3                0.2.6     0.2.7             10.100.200.126:2181   N/A                 11m
+
+```
+Once, the Upgrade completes, output for `$ kubectl get zk` command will have the Version field set as he Desired Version, as shown below
+
 ```
+$ kubectl get zk
+
+NAME         REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT     EXTERNAL ENDPOINT   AGE
+example        3          3               0.2.7     0.2.7            10.100.200.126:2181   N/A                 11m
 
-If the values of the fields `currentRevision` and `updateRevision` are different, it indicates that the `ZookeeperCluster` is currently undergoing an upgrade. The value of `currentRevision` is set to the value of `updateRevision` when the upgrade is complete.
 
 ```
-Note: The value of the tag field should not be modified while an upgrade is already in progress.
+Additionally, the describe output will have the Upgrading status set to False and PodsReady status set to True which signifies that the upgrade has completed, as shown below
```

Comment:
Additionally, the `Upgrading` status set to False and `PodsReady` status set to True, which signifies that the upgrade has completed, as shown below

Reply:
I have changed it.